plugins/signals-backend/src/service/SignalManager.ts

```diff
@@ -64,6 +68,46 @@ export class SignalManager {
       onEvent: (params: EventParams) =>
         this.onEventBrokerEvent(params.eventPayload as SignalPayload),
     });
+
+    if (
+      options.config.getOptionalBoolean('signals.server.pingEnabled') ??
+      true
+    ) {
+      const pingIntervalConfig = options.config.getOptionalConfig(
+        'signals.server.pingInterval',
+      );
+      this.pingInterval = setInterval(
```

Comment:
What do you think about driving this whole thing from the client instead? Have each client have an interval and timeouts where they expect a response, etc. Then on the server-side all we need is really a single interval that loops through all connections and makes sure that the most recently received ping isn't too far in the past.

Reply:
I think it would make sense to have this in both ends. Just left that out for now as I couldn't figure out how to do ping message with plain browser WebSocket (https://stackoverflow.com/questions/10585355/sending-websocket-ping-pong-frame-from-browser). The ws server though automatically responds to ping messages (like browsers do too) and depending on browser they may send these frames automatically. 

One opportunity would be to send the ping/pong as plain message but I think it's too much overhead.